webui/react/src/components/kit/useTags.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { useState } from 'react';
+
+import Tags, { Props, tagsActionHelper } from 'components/kit/Tags';
+
+export const useTags = (
+  tags: string[],
+): ((props?: Omit<Props, 'onAction' | 'tags'>) => JSX.Element) => {
+  const [ctags, setCtags] = useState<string[]>(tags);
+  const onAction = tagsActionHelper(ctags, (ts) => setCtags(ts));
```

Comment:
follow up to my previous comment, but maybe we just don't need this hook? I'm assuming the purpose is to avoid having to write `tagsActionHelper` every time you use the component, but then my thought would be: if we can encapsulate that in a hook, why not just do it all in the component itself?